Common/Systems/Affixes/Affix.cs

```diff
@@ -173,6 +174,10 @@ public static ItemAffix FromBReader(BinaryReader reader)
 
 		return resultList;
 	}
+
+	internal virtual void CreateLocalization()
```

Comment:
Perhaps you should be able to make `Affix` implement `ILocalizedModType`, that will also grant access to extensions such as `this.GetLocalization`, etc.

Reply:
Right, I forgot to do that after I learned of ILocalizedModType, and I didn't know of it when I wrote this.